vagrant/provisioning/roles/pki/tasks/main.yml

```diff
@@ -19,9 +19,15 @@
 - name: generate root ca private key
   become: yes
   openssl_privatekey:
-    path: "/etc/tls/private/arkcase-root.pem"
+    path: "/etc/tls/private/arkcase-root-passphrase.pem"
```

Comment:
these changes fix PKI generation in fips mode.

Reply:
... we also have to make a change in yarn so it doesn't use fips

Reply:
specifically, this patch is needed for yarn on fips, but I'd rather keep this a manual step for now:

```bash
[david.miller_admin@eeocppappaapp01 ~]$ diff -Naurb /usr/share/yarn/lib/cli.js.original /usr/share/yarn/lib/cli.js
--- /usr/share/yarn/lib/cli.js.original 2020-02-20 23:18:25.444000000 +0000
+++ /usr/share/yarn/lib/cli.js  2020-02-20 23:19:05.765000000 +0000
@@ -43202,7 +43202,7 @@
 var stream = __webpack_require__(35);
 
 function hash(content) {
-  var type = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : 'md5';
+  var type = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : 'sha1';
 
   return crypto.createHash(type).update(content).digest('hex');
 }
```